### DISCUSSION TITLE
Farily reliable for combines but not for seeding

### DISCUSSION THREAD

**Discussion by TSO-Original** [Mogli12/AIVehicleExtension#267] [General]
First, this may be my favorite FS19 mod! Thankyou for all your work on it!

I am playing Ravenport multiplayer on my own dedicated server.

I always extend and/or join fields

AI Vehicle Extension is working fairly good for me for combines, but not for seeding.

The seeders go a little way, sometimes, 50-75% of the way around, but then they become confused and start going around in circles etc.

I had wondered if it was due to using a specific tractor mod (big bud) and/or seeder mod (John Deere 1800 series), but it happens with my Giants tractors too. It is bad enough that now I enable AI Vehicle extension for combining, then save game, quit and reboot server with it disabled until my seeding is done.

**Comment by Mogli12** [comment]
Hi,

There is no need to restart a game w/o AIVE to disable it. You can open the settings dialog (KEY_lshift KEY_h) and turn off the alternative worker.

Additionally, you can use the turn mode "straight like normal worker" if the vehicle should go up and down in straight lines.

Regards,
Stefan

**Comment by TSO-Original** [comment]
Thankyou!

I had tried that previously, but since your reply, I discovered that I had not been clicking OK int eh bottom left and so it hadn't been heeding my preferences. It is now working better and I can get the harvesters to unload whithout stopping. Thankyou!

**Comment by Mogli12** [comment]
If there are still problems with the seeder, can you please post some screen shots?
